### ISSUE TITLE
Feature request: insert blank lines between using statement groups

### ISSUE THREAD

**Issue by Pajamaman** [codecadwallader/codemaid#151]
First of all, I love - LOVE - this extension :)

The one thing that seems to be missing is an option to insert blank lines between using statement groups. For instance, I'd like cleanup to change this:

```c#
using System;
using System.Collections.Generic;
using System.Threading;
using DataAccessLayer;
using Foo.Domain;
using Foo.Models;
```

To this:

```c#
using System;
using System.Collections.Generic;
using System.Threading;

using DataAccessLayer;

using Foo.Domain;
using Foo.Models;
```

This feature is only missing in the sense that I'm using CodeMaid instead of ReSharper, and the latter does it :)

**Comment by codecadwallader** [comment]
Thanks for the praise, I really appreciate hearing that. :)

We do have a card for this feature in our backlog.  Check it out and vote for it here: https://trello.com/c/AXK4SaJN

**Comment by hyperquantum** [comment]
Shouldn't this issue be reopened, since the Trello backlog is considered deprecated?

**Comment by codecadwallader** [comment]
Yes, it can be reopened.  When we consolidated Trello into GitHub we brought over the most voted requests and anything that was specifically re-requested.

**Comment by hyperquantum** [comment]
Thanks. I would like to request this feature.
